src/metabase/models/humanization.clj

```diff
@@ -39,10 +40,10 @@
   #{"id" "url" "ip" "uid" "uuid" "guid"})
 
 (defn- capitalize-word [word]
-  (if (contains? acronyms (str/lower-case word))
-    (str/upper-case word)
+  (if (contains? acronyms (u/lower-case-en word))
+    (u/upper-case-en word)
     ;; We are assuming that ALL_UPPER_CASE means we should be Title Casing
-    (if (= word (str/upper-case word))
+    (if (= word (u/upper-case-en word))
```

Comment:
I think changing this is still correct. If your system locale is Turkish and we want to use title casing it would take a column like `id` and display it as`İD`, which probably doesn't make sense. Even if your system locale is Turkish, there's a good chance your database is using ASCII table and column names.

If we do want to take locale into consideration, then we would want to call `(.toUpperCase <str> <locale>)` with the either the user locale or site locale, rather than letting `str/upper-case` fall back to the system locale

Reply:
I suggest adding a function or two for the local specific conversion.

Reply:
If we know that we are lowercasing table names, then this might be indeed fine.

Reply:
From looking at the source, I think the primary use of this is for table and field names. Are we ok sticking with it as-is or do we need further changes?